frontend/components/Course/StudentQueuePage/QuestionCard.tsx

```diff
@@ -47,6 +50,13 @@ const QuestionCard = (props: QuestionCardProps) => {
         });
     };
 
+    useEffect(() => {
+        if (question.status == QuestionStatus.ACTIVE) {
+            play();
+        }
+        // eslint-disable-next-line
+    }, [question.status]);
```

Comment:
ESLint is actually right on this one, you shouldn't disable it, and should add "play" to the dependency array. Remember that play() actually depends on notif, so right now if notif changes play() doesn't get updated properly here and might be stale.